webui/react/src/pages/ExperimentDetails/ExperimentActions.tsx

```diff
@@ -177,7 +164,7 @@ const ExperimentActions: React.FC<Props> = ({ experiment, onClick, trials, onSet
         key="tensorboard"
         loading={btnLoadingStates.Tensorboard}
         onClick={handleCreateTensorboard}>View in TensorBoard</Button>,
-      showIf: (exp): boolean => !experimentWillNeverHaveData(exp, trials),
+      showIf: (exp): boolean => terminalRunStates.has(exp.state),
```

Comment:
blocking: I think we can allow the button to show even when experiment is not in a terminal state.

Reply:
(that is the existing behavior as well)

Reply:
ah yes, I totally did not think this part through. At this point TBs should just be allowed regardless of state.